datalad/utils.py

```diff
@@ -2105,31 +1913,6 @@ def safe_print(s):
 # IO Helpers
 #
 
-# unused in -core
-def open_r_encdetect(fname, readahead=1000):
```

Comment:
FWIW should be good to go. The original/only use was RFed in 5e07eadd76f7dabef81eaa8ec0310d42867af89c